dynamic-simulation/dynamic-simulation-dsl/src/test/java/com/powsybl/dynamicsimulation/groovy/GroovyDynamicModelSupplierTest.java

```diff
@@ -68,4 +68,28 @@ public void test() {
         assertEquals("LOAD", dynamicModel2.getId());
         assertEquals("LOAD", dynamicModel2.getParameterSetId());
     }
+
+    @Test
+    public void testWithInputStream() {
```

Comment:
Maybe you can add a method to share code with previous test?

Reply:
done